test/basic.js

```diff
@@ -763,6 +764,20 @@ test('append alignment to bitfield boundary', async function (t) {
   }
 })
 
+test('setKeyPair', async function (t) {
```

Comment:
(detail) I'd remove this test, since it's not really related to this PR and it's already a very big PR (fine to PR separately though, particularly if we don't have tests for setKeyPair already)

Reply:
Removed